modules/core/04-channel/keeper/packet_test.go

```diff
@@ -354,6 +354,25 @@ func (suite *KeeperTestSuite) TestRecvPacket() {
 			},
 			nil,
 		},
+		{
+			"success with an counterparty latest sequence send set to 0",
+			func() {
+				suite.coordinator.Setup(path)
+				sequence, err := path.EndpointA.SendPacket(defaultTimeoutHeight, disabledTimeoutTimestamp, ibctesting.MockPacketData)
+				suite.Require().NoError(err)
+				packet = types.NewPacket(ibctesting.MockPacketData, sequence, path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID, path.EndpointB.ChannelConfig.PortID, path.EndpointB.ChannelID, defaultTimeoutHeight, disabledTimeoutTimestamp)
+				channelCap = suite.chainB.GetChannelCapability(path.EndpointB.ChannelConfig.PortID, path.EndpointB.ChannelID)
+
+				channel := path.EndpointB.GetChannel()
+				channel.State = types.STATE_FLUSHING
+				path.EndpointB.SetChannel(channel)
+
+				// set last packet sent sequence to zero.
+				counterpartyUpgrade := types.Upgrade{LatestSequenceSend: 0}
```

Comment:
Followed what the rests of the tests are doing which is mutating state directly, tho would be nicer if we'd consistently use the handlers to get here.